bitnami/external-dns/Chart.yaml

```diff
@@ -24,4 +24,4 @@ sources:
   - https://github.com/kubernetes-sigs/external-dns
   - https://github.com/bitnami/bitnami-docker-external-dns
   - https://github.com/kubernetes-sigs/external-dns
-version: 5.1.0
+version: 5.1.1
```

Comment:
New line needed here to pass the lint check

Reply:
I removed the new line because it failed on the lint check, let me reimplement it.

Reply:
This is strange, Github says this file has 28 lines in the [UI view](https://github.com/dza89/charts-4/blob/update-azure-private-dns/bitnami/external-dns/Chart.yaml) but if you download it raw it has 29, so the lint check fails. It may be that your IDE is messing with the endlines, you can verify locally if the file passes the lint check by running `ct lint --config .github/ct-lint.yaml` at the repo root directory

Reply:
I tried it in the github web ide, shouldn't try that anymore it seems. Fixed.